### PR TITLE
add russian translation for readme

### PR DESCRIPTION
**Pull request by highflyer910**
Create Russian translation of README.md

#3106

### CODE REVIEW

File: translations/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) | [:ru: Russian](README.russian.md) |
```
Comment (marionschleifer) [inline translations/README.md]: This doesn't show up correctly:

image

File: translations/README.russian.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+Hasura GraphQL Engine - это сверхбыстрый графический сервер позволяющий создавать **мгновенно GraphQL API на основе Postgres**, с [**webhook triggers**](event-triggers.md), и [**remote schemas**](remote-schemas.md) для бизнес-логики.
+
+Hasura помогает стройть GraphQL приложения при поддержке Postgres или постепенно перейти на GraphQL для уже существующих приложений, использующих Postgres.
+
+Подробнее на [hasura.io](https://hasura.io) и [docs](https://docs.hasura.io).
+
+------------------
+
+![Hasura GraphQL Engine Demo](assets/demo.gif)
```
Comment (marionschleifer) [inline translations/README.russian.md]: This doesn't show up correctly: 

image

File: translations/README.russian.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+* **Интерфейс администрирования и миграции**: Интерфейс администрирования и миграция схемы на Rails
+* **Postgres** ❤️: Поддержка Postgres типов (PostGIS/geo-location, и т.д.), преобразуется в *graphs*, запускает процедуры или функции, сохраненные посредством мутаций.
+
+Подробнее на [hasura.io](https://hasura.io) и [docs](https://docs.hasura.io).
+
+## Содержание
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Содержание**
+
+- [Быстрый запуск:](#quickstart)
```
Comment (marionschleifer) [inline translations/README.russian.md]: Links don't work here.

**Comment by highflyer910** [comment]
Fixed everything:)

File: translations/README.russian.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+* **Интерфейс администрирования и миграции**: Интерфейс администрирования и миграция схемы на Rails
+* **Postgres** ❤️: Поддержка Postgres типов (PostGIS/geo-location, и т.д.), преобразуется в *graphs*, запускает процедуры или функции, сохраненные посредством мутаций.
+
+Подробнее на [hasura.io](https://hasura.io) и [docs](https://docs.hasura.io).
+
+## Содержание
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Содержание**
+
+- [Быстрый запуск:](#быстрый_запуск)
```
Comment (marionschleifer) [inline translations/README.russian.md]: Some of the links in the table of contents are still broken.

File: translations/README.russian.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+Идеально подходит для уведомлений, каналов передачи данных от Postgres или асинхронной обработки - [подробнее](../event-triggers.md).
+
+### Полученные данные или трансформация данных
+
+Преобразование данных в Postgres или запуск бизнес-логики для получения другого набора данных, который можно запросить с помощью GraphQL Engine. - [подробнее](https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html).
+
+## Demos
+
+Ознакомьтесь со всеми примерами приложений в 
+[community/examples](../community/examples) папке.
```
[the remark this answers is not in the document]
Comment (highflyer910) [reply translations/README.russian.md]: Fixed and found the same issue on the main readme file, may I fix it too?

File: translations/README.russian.md (hunk 4 above)
Comment (highflyer910) [reply translations/README.russian.md]: Done!

File: translations/README.russian.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (marionschleifer) [reply translations/README.russian.md]: That would be great! 🙌

File: translations/README.russian.md (hunk 4 above)
Comment (marionschleifer) [reply translations/README.russian.md]: The very first link in the table of contents is still not working. Everything else works 🙂

File: translations/README.russian.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (highflyer910) [reply translations/README.russian.md]: Fixed:)

File: translations/README.russian.md (hunk 4 above)
Comment (highflyer910) [reply translations/README.russian.md]: Fixed the 1st link too. What silly mistakes I do today!:)

**Comment by marionschleifer** [comment]
@highflyer910 could you please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

File: translations/README.russian.md [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+[![CircleCI](https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)](https://circleci.com/gh/hasura/graphql-engine)
+
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine - это сверхбыстрый графический сервер позволяющий создавать **мгновенный GraphQL API работающий в режиме реального времени в базе данных Postgres**, с [**webhook триггерами**](../event-triggers.md) на основе событий в базе данных, и [**remote schemas**](../remote-schemas.md) для бизнес-логики.
+
+Hasura помогает стройть GraphQL приложения при поддержке Postgres или постепенно перейти на GraphQL для уже существующих приложений, использующих Postgres.
```
Comment (snelsi) [inline translations/README.russian.md]: fix a typo
```suggestion
Hasura помогает строить GraphQL приложения при поддержке Postgres или постепенно перейти на GraphQL для уже существующих приложений, использующих Postgres.
```

**Comment by highflyer910** [comment]
@marionschleifer Done :)

File: translations/README.russian.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+## Контрибуция
+
+Ознакомьтесь с нашим [руководством по контрибуции](../CONTRIBUTING.md) для более подробной информации.
+
+## Элементы бренда
+
+Элементы бренда Hasura (логотипы, Hasura значки и т.д.) можно найти в [assets/brand](../assets/brand) папке.  Можете использовать их в вашем приложении / веб-сайте и т.д. 
+Мы были бы рады, если бы вы добавили "Powered by Hasura" бейдж к вашим приложениям, построенным с помощью Hasura.. ❤️
+
+<div style="display: flex;">
```
Comment (marionschleifer) [inline translations/README.russian.md]: The Hasura logo doesn't show properly here:

image

Comment (highflyer910) [reply translations/README.russian.md]: Done finally (I hope:D)

**Comment by shahidhk** [comment]
Hello @highflyer910, can you sign the CLA? https://cla-assistant.io/hasura/graphql-engine?pullRequest=3186

**Comment by highflyer910** [comment]
@shahidhk Done! :)

**Comment by shahidhk** [comment]
@highflyer910 The CLA assistant still says it is not signed: https://github.com/hasura/graphql-engine/pull/3186#issuecomment-544257646

**Comment by shahidhk** [comment]
@highflyer910 Nevermind, thanks for your contribution. :)

**Pull request merged**